string name
uint8[] actual
uint8[] candidate
---
bool accepted
